File: U3P03-PHP-Catalogo/Autor.php
<?php
Class Autor{
    private $cod_DJ;
    private $nombre_dj;
    private $nacionalidad;
    private $descripcion;
    private $genero_musical;
    private $img_autor;
    public function getCod_DJ()
    {
        return $this->cod_DJ;
    }
    public function getNombre_dj()
    {
        return $this->nombre_dj;
    }
    public function getNacionalidad()
    {
        return $this->nacionalidad;
    }
    public function getDescripcion()
    {
        return $this->descripcion;
    }
    public function getGenero_musical()
    {
        return $this->genero_musical;
    }

    public function getImg_autor()
    {
        return "<img src='img/$this->img_autor' height=100px width=90px>";
    } 
}


?>

File: U5P02-Java-Catalogo/src/clases/Autor.java
package clases;

public class Autor {
	private int cod_dj;
	private String nombre_dj;
	private String nacionalidad;
	private String descripcion;
	private String genero_musical;
	private String img_autor;
	public Autor(int cod_dj, String nombre_dj, String nacionalidad, String descripcion, String genero_musical,
			String img_autor) {
		super();
		this.cod_dj = cod_dj;
		this.nombre_dj = nombre_dj;
		this.nacionalidad = nacionalidad;
		this.descripcion = descripcion;
		this.genero_musical = genero_musical;
		this.img_autor = img_autor;
	}
	public int getCod_dj() {
		return cod_dj;
	}
	public void setCod_dj(int cod_dj) {
		this.cod_dj = cod_dj;
	}
	public String getNombre_dj() {
		return nombre_dj;
	}
	public void setNombre_dj(String nombre_dj) {
		this.nombre_dj = nombre_dj;
	}
	public String getNacionalidad() {
		return nacionalidad;
	}
	public void setNacionalidad(String nacionalidad) {
		this.nacionalidad = nacionalidad;
	}
	public String getDescripcion() {
		return descripcion;
	}
	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}
	public String getGenero_musical() {
		return genero_musical;
	}
	public void setGenero_musical(String genero_musical) {
		this.genero_musical = genero_musical;
	}
	public String getImg_autor() {
		return img_autor;
	}
	public void setImg_autor(String img_autor) {
		this.img_autor = img_autor;
	}
	

}
